molgenis-web/src/main/java/org/molgenis/web/exception/GlobalControllerExceptionHandler.java

```diff
@@ -0,0 +1,83 @@
+package org.molgenis.web.exception;
+
+import org.molgenis.data.UnknownDataException;
+import org.molgenis.data.security.exception.EntityTypePermissionDeniedException;
+import org.molgenis.data.validation.DataIntegrityViolationException;
+import org.molgenis.data.validation.ValidationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.web.HttpRequestMethodNotSupportedException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.method.HandlerMethod;
+import org.springframework.web.servlet.NoHandlerFoundException;
+
+import javax.servlet.http.HttpServletRequest;
+
+import static org.molgenis.web.exception.ExceptionHandlerUtils.handleTypedException;
+import static org.molgenis.web.exception.ExceptionHandlerUtils.isHtmlRequest;
+import static org.springframework.http.HttpStatus.*;
+
+@ControllerAdvice
+@Order(Ordered.HIGHEST_PRECEDENCE)
+public class GlobalControllerExceptionHandler
```

Comment:
We need to decide which exceptions the Global Exception Handler should know about.